frontend/public/components/factory/Table/VirtualizedTable.tsx

```diff
@@ -224,7 +226,8 @@ const VirtualizedTable: VirtualizedTableFC = ({
             )}
             <PfTable
               cells={columns}
-              rows={[]}
+              rows={[{ selected: allRowsSelected }]}
```

Comment:
I think this will break if canSelectAll is false, right?

Reply:
It does not break. I can't see the checkbox in the header